src/main/java/com/tabnine/balloon/GotItTooltip.java

```diff
@@ -0,0 +1,116 @@
+package com.tabnine.balloon;
+
+import static com.tabnine.general.DependencyContainer.instanceOfBinaryRequestFacade;
+import static com.tabnine.general.StaticConfig.ICON;
+import static com.tabnine.general.Utils.wrapWithHtml;
+import static com.tabnine.general.Utils.wrapWithHtmlTag;
+
+import com.intellij.openapi.Disposable;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.ui.popup.Balloon;
+import com.intellij.openapi.ui.popup.JBPopupFactory;
+import com.intellij.openapi.ui.popup.JBPopupListener;
+import com.intellij.openapi.ui.popup.LightweightWindowEvent;
+import com.intellij.openapi.util.Disposer;
+import com.intellij.ui.JBColor;
+import com.intellij.ui.awt.RelativePoint;
+import com.intellij.util.ui.JBUI;
+import com.tabnine.binary.BinaryRequestFacade;
+import com.tabnine.binary.requests.notifications.shown.HintShownRequest;
+import javax.swing.*;
+import org.jetbrains.annotations.NotNull;
+
+public class GotItTooltip implements Disposable {
```

Comment:
rename class to `SuggestionHintTooltipUI` or something

Reply:
the purpose of this class is to provide other hints we may use in the future, and not just the completion hint